include/swift/AST/Module.h

```diff
@@ -209,7 +210,20 @@ class ModuleDecl : public DeclContext, public TypeDecl {
 
 public:
   typedef ArrayRef<Located<Identifier>> AccessPathTy;
-  typedef std::pair<ModuleDecl::AccessPathTy, ModuleDecl*> ImportedModule;
+  
+  typedef struct _ImportedModule {
```

Comment:
If you look at the rest of the codebase, most (all?) of it uses C++ style declaration of structs.

```
struct Typename {
  // body
};

Typename Fieldname;
```

Could you follow the same style here?

I guess the question would be what to name the struct vs the field. If you look at the rest of this file, it seems to use lowercase names for fields. For example, we have code like

```
   Optional<std::pair<ModuleDecl *, Identifier>> declaringModuleAndBystander;
```

later in this file. Following that, you could name the type `ImportedModule` (no underscores) and the field `importedModule`.

Reply:
Resolved in af9a745